modules/microshift-get-cluster-id-kubesystem.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// microshift_support/microshift-getting-cluster-id.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-get-cluster-id-kubesystem_{context}"]
+= Getting your cluster ID using the kube-system namespace
+
+.Procedure
+
+. Get the namespace UID of the cluster which is running by entering the following command:
++
+[source,terminal]
+----
+$ oc get namespaces kube-system -o jsonpath={.metadata.uid}
+----
+
+. Get the namespace UID of a cluster which is not running by entering the following command:
++
+[source,terminal]
+----
+$ sudo cat /var/lib/microshift/cluster-id
+----
+
```

Comment:
```suggestion
+
```
Looks like the admonition is restarting the numbering. If you tuck it in, the third step should number correctly as "3."

Reply:
done